modules/AOS_PDF_Templates/templateParser.php

```diff
@@ -92,7 +92,11 @@ function parse_template_bean($string, $key, &$focus)
                     $link = $secureLink;
                     $repl_arr[$key . "_" . $fieldName] = '<img src="' . $link . '" width="'.$field_def['width'].'" height="'.$field_def['height'].'"/>';
                 } else {
-                    $repl_arr[$key . "_" . $fieldName] = $focus->$fieldName;
+                    $repl_arr[$key . "_" . $fieldName] = (isset($focus->$fieldName) ? $focus->$fieldName : false);
+
+                    if (!isset($focus->$fieldName)) {
+                        LoggerManager::getLogger()->warn('PDF Template property does not exist');
```

Comment:
It's cool. If you concatenate the `$fieldName` into the error message makes it more sence.